### ISSUE TITLE
vcs_git_download_url does not support when using parameters  

### ISSUE THREAD

**Issue by eliorso** [jfrog/terraform-provider-artifactory#780]
**Describe the bug**
We had an issue with bower remote repo which did not works, the support ( ticket 260432 ) advised us to configure vcs_git_download_url as follow: "{0}/{1}/archive/{2}.{3}" but we are getting the following error : 
```Error: expected "vcs_git_download_url" to have a host, got {0}/{1}/archive/{2}.{3}

Our terrafrom code: 
      bower = {
         "bower-wix-private" = {
              url                                   = "https://github.com/"
              vcs_git_provider                      = "CUSTOM"
              bower_registry_url                    = "https://github.com/"
              vcs_git_download_url                  = "{0}/{1}/archive/{2}.{3}"
              username                              = "********"
              password                              = ********
              unused_artifacts_cleanup_period_hours = 48
              content_synchronisation = {
                enabled                         = true
                statistics_enabled              = true
                properties_enabled              = true
                source_origin_absence_detection = true
          }
        }

Module conf : 

resource "artifactory_remote_bower_repository" "bower-remote" {
  for_each = try(var.repositories.remote.bower, {})


  key                  = each.key
  project_key          = lookup(each.value, "project_key", "default")
  url                  = lookup(each.value, "url", null)
  xray_index           = lookup(each.value, "xray_index", null)
  username             = lookup(each.value, "username", null)
  password             = lookup(each.value, "password", null)
  vcs_git_provider     = lookup(each.value, "vcs_git_provider", "GITHUB")
  bower_registry_url   = lookup(each.value, "bower_registry_url", "https://registry.bower.io")
  vcs_git_download_url = null
  #vcs_git_download_url                  = lookup(each.value, "vcs_git_download_url", null)
  unused_artifacts_cleanup_period_hours = lookup(each.value, "unused_artifacts_cleanup_period_hours", null)

  content_synchronisation {
    enabled                         = try(lookup(each.value.content_synchronisation, "enabled", null), null)
    properties_enabled              = try(lookup(each.value.properties_enabled, "enabled", null), null)
    source_origin_absence_detection = try(lookup(each.value.source_origin_absence_detection, "enabled", null), null)
    statistics_enabled              = try(lookup(each.value.statistics_enabled, "enabled", null), null)
  }

}

**Comment by danielmkn** [comment]
Hi @eliorso! Can you please provide an actual error message? I don't see it in the ticket description.

**Comment by alexhung** [comment]
Also, please provide the information for a bug ticket:

> **Describe the bug**
> A clear and concise description of what the bug is.
> 
> **Requirements for and issue**
> - [ ] A description of the bug
> - [ ] A fully functioning terraform snippet that can be copy&pasted (no outside files or ENV vars unless that's part of the issue). **If this is not supplied, this issue will likely be closed without any effort expended.**
> - [ ] Your version of artifactory (you can `curl` it at `$host/artifactory/api/system/version`
> - [ ] Your version of terraform
> - [ ] Your version of terraform provider
>

**Comment by eliorso** [comment]
Hi, 

I just see that the error was not added. 

The error we can see from Terraform is: 
` Error: expected "vcs_git_download_url" to have a host, got {0}/{1}/archive/{2}.{3} `

We are working with Artifactory SaaS.

provider version : 
    artifactory = {
      source  = "registry.terraform.io/jfrog/artifactory"
      version = "7.5.0"
      

Terrafrom version :  "1.4.6"

**Comment by alexhung** [comment]
@eliorso This is fixed in [v8.2.2](https://github.com/jfrog/terraform-provider-artifactory/releases/tag/v8.2.2). If you can upgrade to this version, this is preferred solution 😄
